Recruit-Training/Advanced-Recruit-Training/src/BatteryStateOfChargeService.h

```diff
@@ -25,4 +25,12 @@ class BatteryStateOfChargeService : public I_BatteryStateOfChargeService
     const double initialStateOfChargePercent_;
 
     // Add your necessary variables here
+    bool isCharging_;
+    double prevCurr;
```

Comment:
Team convention is to leave a trailing underscore at the end of a private variable.